src/webui/www/public/scripts/dynamicTable.js

```diff
@@ -253,6 +253,10 @@ var dynamicTable = new Class({
                     if ((state != 'downloading') && !~state.indexOf('DL'))
                         return false;
                     break;
+                case 'seeding':
+                    if ((state != 'uploading') && (state != 'stalledUP') && (state != 'queuedUP'))
```

Comment:
I thought the same, but that's not what's happening in the regular UI. If a torrent is completed or seeding and I force a recheck, it's not longer seeding and completed but inactive.